ckanext/canada/tests/__init__.py

```diff
@@ -5,15 +5,18 @@
     tables_exist as validation_tables_exist
 )
 from ckanext.security.model import db_setup as security_db_setup
+from ckan.cli.db import _run_migrations
 
 class CanadaTestBase(object):
     @classmethod
     def setup_method(self, method):
         """Method is called at class level before EACH test methods of the class are called.
         Setup any state specific to the execution of the given class methods.
         """
+        _run_migrations('canada_public')
```

Comment:
It should be possible to use the with_plugins and clean_db fixtures (like the activity plugin tests do) instead of running migrations directly

Reply:
@wardi yeah, so after a lot of debugging, the database in the Docker Image for CKAN is always going to be ahead of the migrations we have. That is also why I have to call `validation_create_tables` and `security_db_setup` in the test base class too.